process/interface.go

```diff
@@ -67,6 +67,13 @@ type TxValidatorHandler interface {
 	Fee() *big.Int
 }
 
+// TxVersionCheckerHandler defines the functionality that is needed for a TxVersionChecker to validate transaction version
+type TxVersionCheckerHandler interface {
+	IsSignedWithHash(tx *transaction.Transaction) bool
```

Comment:
Should use the data.TransactionHandler when defining IsSignedWithHash and CheckTxVersion methods

Reply:
If i use TransactionHandler I don't have expose field Version and Option and I'm not sure if all structures that implements this interface needs to have that fields.

Reply:
ok